pkg/version/channels.go

```diff
@@ -73,6 +80,9 @@ func getLatestVersions(ctx context.Context, client *http.Client, url string) (Ch
 
 	rsp, err := client.Do(req.WithContext(ctx))
 	if err != nil {
+		if errors.As(err, &DnsError) {
```

Comment:
we could define the `var DnsError` right here, local to this scope, instead of as a package scoped var.  That makes it clear that it's just used for error matching here rather than being used to communicate some kind of global state.